### PR TITLE
Feature/search profiles

### PR DESCRIPTION
**Pull request by harshvitra**
Draft PR for search profiles

Check the CORS error

### CODE REVIEW

**Comment by norbertschuler** [comment]
There is some code in the develop branch to add headers at https://github.com/Plant-for-the-Planet-org/treecounter-platform/blob/61a0ae330a556eb34667a3b1d7a078d7843244a5/public/suggest.php#L3-L6, but currently I do not get these headers when calling the script in the development environment e.g. with curl -I https://app-development.plant-for-the-planet.org/suggest - that's why the CORS errors appear:
```
HTTP/2 307 
date: Thu, 07 Jan 2021 09:37:50 GMT
content-type: text/html
set-cookie: __cfduid=d965e0408969d8bc79e91d98d8acb0ff41610012269; expires=Sat, 06-Feb-21 09:37:49 GMT; path=/; domain=.plant-for-the-planet.org; HttpOnly; SameSite=Lax
location: https://app-development.plant-for-the-planet.org/suggest.php
cf-ray: 60dcb3cf59614169-HAM
via: 1.1 vegur
cf-cache-status: DYNAMIC
cf-request-id: 077dceb593000041696c9b0000000001
expect-ct: max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"
server: cloudflare

```

**Comment by norbertschuler** [comment]
If we call the PHP-Script directly, the headers are correctly set: curl -I https://app-development.plant-for-the-planet.org/suggest.php - they won't be set for https://app-development.plant-for-the-planet.org/suggest as that's a redirect specified in the Heroku Nginx config not setting these headers at https://github.com/Plant-for-the-Planet-org/treecounter-platform/blob/61a0ae330a556eb34667a3b1d7a078d7843244a5/nginx_app.conf#L7-L10
```
HTTP/2 200 
date: Thu, 07 Jan 2021 09:53:32 GMT
content-type: application/json
set-cookie: __cfduid=de06a433172edc5ee07d2cdf7a70d57b11610013211; expires=Sat, 06-Feb-21 09:53:31 GMT; path=/; domain=.plant-for-the-planet.org; HttpOnly; SameSite=Lax
cf-ray: 60dccace7851414b-HAM
access-control-allow-origin: *
cache-control: no-cache, private
via: 1.1 vegur
cf-cache-status: DYNAMIC
access-control-allow-headers: *
access-control-allow-methods: GET, POST, PUT, DELETE, OPTIONS
cf-request-id: 077ddd150f0000414b4f900000000001
expect-ct: max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"
server: cloudflare

```

**Comment by norbertschuler** [comment]
Now the results are getting delivered and need to be processed :-), e.g.:

```json
[
  {
    "treecounterId": 124188,
    "id": 103416,
    "slug": "norbert-schuler",
    "name": "Norbert Schuler",
    "image": "5fdccb635fbb6992671679.jpeg",
    "type": "individual",
    "mayPublish": true,
    "category": "profile"
  },
  {
    "treecounterId": 174004,
    "id": 256477,
    "slug": "norberts-baumschule",
    "name": "Norberts Baumschule",
    "type": "tpo",
    "mayPublish": true,
    "category": "profile"
  }
]
```

**Review by norbertschuler — APPROVED** [review]
LGFM

File: src/tenants/planet/LeaderBoard/components/Score.tsx [hunk 1]
```diff
@@ -41,6 +72,49 @@ export default function LeaderBoardSection(leaderboard: Props) {
...
+                    />
+                    <span>{option.name}</span>
+                    </div>
+
+                </Link>
+              )}
+              renderInput={(params) => (
+                <MaterialTextField
+                  {...params}
+                  label="Search User"
```
Comment (norbertschuler) [inline src/tenants/planet/LeaderBoard/components/Score.tsx:105]: Need a translation resource here if we want to keep this UI design.

**Comment by sagararyal** [comment]
Still getting CORS on Safari.

Screen Shot 2021-01-13 at 1 20 03 PM

**Comment by norbertschuler** [comment]
> Still getting CORS on Safari, Firefox

And this won't go away until the backend return a proper header at staging - test with `curl -I https://app-staging.plant-for-the-planet.org/suggest.php` vs. `curl -I https://app-development.plant-for-the-planet.org/suggest.php`.

**Pull request merged**